### ISSUE TITLE
listData became an unnamed list

### ISSUE THREAD

**Issue by frymor** [lawremi/rtracklayer#18]
Hi, 
When importing a bigwig file using the `import.bw` command I ususally get a named list in my `listData` slot of the object. 
But in my latest version (1.44.0), the slot contains an unnamed list.

can you please correct it? Or is there a special reason for it to be unnamed?

thanks
Assa

In rtracklayer version **1.38.3** the objkect has the names of the chromosome in a list

```
> str(bigWigList[[1]])
Formal class 'SimpleNumericList' [package "IRanges"] with 4 slots
...
  ..@ listData       :List of 17
  .. ..$ chrI   : num [1:230218] 0 1 1 1 1 2 4 6 8 10 ...
  .. ..$ chrII  : num [1:813184] 8 14 24 30 37 44 56 58 64 67 ...
  .. ..$ chrIII : num [1:316620] 1 4 5 7 7 10 11 10 15 18 ...
  .. ..$ chrIV  : num [1:1531933] 7 7 15 17 20 22 24 30 34 37 ...
  .. ..$ chrIX  : num [1:439888] 15 42 69 87 98 111 127 146 161 185 ...
  .. ..$ chrV   : num [1:576874] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ chrVI  : num [1:270161] 0 3 3 4 6 8 10 11 11 11 ...
  .. ..$ chrVII : num [1:1090940] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ chrVIII: num [1:562643] 4 4 4 4 4 4 4 4 4 4 ...
  .. ..$ chrX   : num [1:745751] 16 42 55 70 82 102 114 120 132 142 ...
  .. ..$ chrXI  : num [1:666816] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ chrXII : num [1:1078177] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ chrXIII: num [1:924431] 0 1 1 1 1 1 1 1 1 1 ...
  .. ..$ chrXIV : num [1:784333] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ chrXV  : num [1:1091291] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ chrXVI : num [1:948066] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ chrmt  : num [1:85779] 33 52 70 71 73 80 83 84 85 87 ...
```
But in a different session with rtracklayer version **1.44.0** I don't have them anymore.
```
 str(bigWigList[[1]])
...
  ..@ listData       :List of 17
  .. ..$ : num [1:230218] 0 1 1 1 1 2 4 6 8 10 ...
  .. ..$ : num [1:813184] 8 14 24 30 37 44 56 58 64 67 ...
  .. ..$ : num [1:316620] 1 4 5 7 7 10 11 10 15 18 ...
  .. ..$ : num [1:1531933] 7 7 15 17 20 22 24 30 34 37 ...
  .. ..$ : num [1:439888] 15 42 69 87 98 111 127 146 161 185 ...
  .. ..$ : num [1:576874] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ : num [1:270161] 0 3 3 4 6 8 10 11 11 11 ...
  .. ..$ : num [1:1090940] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ : num [1:562643] 4 4 4 4 4 4 4 4 4 4 ...
  .. ..$ : num [1:745751] 16 42 55 70 82 102 114 120 132 142 ...
  .. ..$ : num [1:666816] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ : num [1:1078177] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ : num [1:924431] 0 1 1 1 1 1 1 1 1 1 ...
  .. ..$ : num [1:784333] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ : num [1:1091291] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ : num [1:948066] 0 0 0 0 0 0 0 0 0 0 ...
  .. ..$ : num [1:85779] 33 52 70 71 73 80 83 84 85 87 ...
```

**Comment by lawremi** [comment]
Please provide a reproducible example.

**Comment by frymor** [comment]
Here is an example

```
library(rtracklayer)
bwFile <- import.bw(con = "http://genome.ucsc.edu/goldenPath/help/examples/bigWigExample.bw", as="NumericList")
```

with version 1.38.0 it looks like that:

```
> bwFile
NumericList of length 1
[["chr21"]] 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 ... 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
```

but with the new version

```
> bwFile
NumericList of length 1
[[1]] 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 ... 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
```

**Comment by lawremi** [comment]
Should be fixed in 1.44.1 (release) or 1.45.2 (devel). Thanks!
